src/components/DefaultNavigationBar.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <q-header
+    class="bg-primary text-white shadow-1 row justify-between items-center q-px-lg"
+    data-cy="navigation-bar"
+  >
+    <div class="application-name">
+      <q-icon
+        color="primary"
+        size="4rem"
+        name="img:icons/logo_modelizer.svg"
+        :left="true"
+        class="home-logo"
+      />
+      <label class="icon-label vertical-middle">
+        {{ $t('application.name') }}
+      </label>
+    </div>
+  </q-header>
+</template>
+
+<style lang="scss" scoped>
+.icon-label {
+  cursor: pointer;
+  color: white;
+  font-size: medium;
```

Comment:
You can use these helper classes from Quasar : 
`class="cursor-pointer text-white"`
And instead of `font-size:medium` see https://quasar.dev/style/typography

Reply:
done